### PR TITLE
mshv: Add integration test for mshvlog service

### PR DESCRIPTION
**Pull request by praveen-pk**
Add an integration test for mshvlog service. If mshv_diag module is loaded, mshlog.service should run without any errors.

Move tests in mshv_root_tests.py to appropriately named mshv_root_stress_tests.py file.

### CODE REVIEW

File: microsoft/testsuites/mshv/mshv_root_tests.py [hunk 1]
```diff
@@ -1,223 +1,87 @@
...
+        With mshv_diag module loaded, ensure mshvlog.service starts and runs
+        successfully on MSHV root partitions. Also confirm there are no errors
+        reported by mshv_diag module in dmesg.
         """,
         priority=4,
-        timeout=10800,  # 3 hours
+        timeout=30,  # 30 seconds
     )
-    def stress_mshv_vm_create(
+    def mshvlog_is_active(
```
Comment (squirrelsc) [inline microsoft/testsuites/mshv/mshv_root_tests.py]: Test case name should start with verify_ if it's a functional test case.

Comment (praveen-pk) [reply microsoft/testsuites/mshv/mshv_root_tests.py]: I will fix it.

Comment (praveen-pk) [reply microsoft/testsuites/mshv/mshv_root_tests.py]: done!

**Comment by squirrelsc** [comment]
@LiliDeng LGTM

**Review by LiliDeng — APPROVED** [review]

**Comment by LiliDeng** [comment]
stress_mshv_vm_create run fail, discuss internally

**Review by LiliDeng — CHANGES_REQUESTED** [review]
stress_mshv_vm_create failed

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**